tests/test_statistics.py

```diff
@@ -436,6 +436,15 @@ def test_univariate_count(self, x):
         ecdf = ECDF(stat="count")
         stat, vals = ecdf(x)
 
+        assert_array_equal(vals[1:], np.sort(x))
+        assert_array_almost_equal(stat[1:], np.linspace(0, 100, len(x) + 1)[1:])
+        assert stat[0] == 0
+
+    def test_univariate_percent(self, x):
+
+        ecdf = ECDF(stat="percent")
+        stat, vals = ecdf(x)
+
```

Comment:
This diff changes the assertion for the count test and then adds a new setup for the percent test but uses the old count test assertion. I think it's only passing by accident because there are 100 entries in the test data. (Probably that is a bad idea and a less-round number should be used for the data fixture to avoid edge cases like this).